io/io/src/TMemFile.cxx

```diff
@@ -113,12 +113,12 @@ TMemFile::EMode TMemFile::ParseOption(Option_t *option)
 }
 
 ////////////////////////////////////////////////////////////////////////////////
-/// Constructor to create a TMemFile re-using external storage.
+/// Constructor to create a TMemFile re-using external C-Style storage.
 
-TMemFile::TMemFile(const char *path, ExternalDataPtr_t data) :
-   TFile(path, "WEB", "read-only TMemFile", 0 /*compress*/),
-   fBlockList(reinterpret_cast<UChar_t*>(const_cast<char*>(data->data())), data->size()),
-   fExternalData(std::move(data)), fSize(fExternalData->size()), fSysOffset(0), fBlockSeek(nullptr), fBlockOffset(0)
```

Comment:
@Axel-Naumann, why we do `std::move(data)`? Isn't that making nullptr the calling pointer?

Reply:
But `data` is a copy of whatever the caller passed: no `const &`.